### PR TITLE
proposed tweaks for GO-space def

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

File: properties/P000154.md [hunk 1]
```diff
@@ -13,12 +13,17 @@ refs:
...
 
-The space $X$ (with topology $\tau$) is {P2} and there is a linear order $<$ on $X$ such that every point has a local base of $\tau$-open neighborhoods consisting of order-convex sets.  (A subset $A$ of an ordered set $(X,<)$ is *order-convex* if for each $a,b\in A$ with $a<b$, $A$ contains all points of $X$ between $a$ and $b$.)
+The space $X$ is {P2} and there is a linear order $<$ on $X$ such that every point
+has a local base of order-convex neighborhoods.
+(A subset $A$ of an ordered set $(X,<)$ is *order-convex* if for each $a,b\in A$ with $a<b$,
+$A$ contains all points of $X$ between $a$ and $b$.)
 
-Equivalently, there is a linear order $<$ on $X$ such that the topology $\tau$ on $X$ contains the order topology induced by the order $<$ and every point has a local base of $\tau$-open neighborhoods consisting of order-convex sets.
+Equivalently, there is a linear order $<$ on $X$ such that intervals $(a,b)=\{x:a<x<b\}$ are open,
+and every point has a local base order-convex neighborhoods.
```
Comment (StevenClontz) [inline properties/P000154.md:22]:
```suggestion
and every point has a local base of order-convex neighborhoods.
```

File: properties/P000154.md [hunk 2]
```diff
@@ -13,12 +13,17 @@ refs:
...
 ---
 
-The space $X$ (with topology $\tau$) is {P2} and there is a linear order $<$ on $X$ such that every point has a local base of $\tau$-open neighborhoods consisting of order-convex sets.  (A subset $A$ of an ordered set $(X,<)$ is *order-convex* if for each $a,b\in A$ with $a<b$, $A$ contains all points of $X$ between $a$ and $b$.)
+The space $X$ is {P2} and there is a linear order $<$ on $X$ such that every point
+has a local base of order-convex neighborhoods.
+(A subset $A$ of an ordered set $(X,<)$ is *order-convex* if for each $a,b\in A$ with $a<b$,
+$A$ contains all points of $X$ between $a$ and $b$.)
 
-Equivalently, there is a linear order $<$ on $X$ such that the topology $\tau$ on $X$ contains the order topology induced by the order $<$ and every point has a local base of $\tau$-open neighborhoods consisting of order-convex sets.
+Equivalently, there is a linear order $<$ on $X$ such that intervals $(a,b)=\{x:a<x<b\}$ are open,
```
Comment (StevenClontz) [inline properties/P000154.md:21]:
```suggestion
Equivalently, there is a linear order $<$ on $X$ such that rays $(a,\rightarrow)=\{x:a<x\}$
and $(\leftarrow,b)=\{x:x<b\}$ are open,
```

**Comment by prabau** [comment]
I had initially phrased the first two definitions in term of a local base of _open_ nbhds of a point as I was worried it was required.  But it seems that a local base of order-convex nbhds (without requiring open) is good enough.  Please check the section "Alternate characterizations" that I added to https://math.stackexchange.com/questions/4917398 to see if that makes sense.

So given that, I like your more concise phrasing.  Approving this, and letting you merge.

**Review by prabau — APPROVED** [review]

**Comment by StevenClontz** [comment]
Ah, well, my definition of "local base" is a collection of open sets. :-) I'd use the term "neighborhood base" to describe what I think you're using "local base" to mean: for every neighborhood $N$ of the point $x$, there exists a neigrborhood $M$ of $x$ in the neighborhood base such that $M\subseteq N$. Fortunately, the point is moot here I think.

**Comment by prabau** [comment]
Yeah, different authors may use different terminology.  "Local base" and "neighborhood base" are synonyms in wikipedia and in Encyclopedia of General Topology (Hart, Nagata, Vaughan eds.).  Willard uses "neighborhood base" and does not use "local base".  But all three mean what you said above (with nbhds not assumed to be open).

**Pull request merged**